.github/workflows/release.yaml

```diff
@@ -28,5 +28,5 @@ jobs:
           body: ${{steps.build_changelog.outputs.changelog}}
           prerelease: "${{ contains(github.ref, '-rc') }}"
           # Ensure target branch for release is "master"
-          commit: master
+          commit: main
```

Comment:
This might break at some point but since cruft uses three-way merge should be fine for a while.

Reply:
I will address this in the next update round